cachito/workers/config.py

```diff
@@ -26,6 +26,9 @@ class DevelopmentConfig(Config):
     cachito_archives_dir = os.path.join(tempfile.gettempdir(), 'cachito-archives')
     cachito_log_level = 'DEBUG'
 
+    result_backend = 'rpc'
```

Comment:
Cool, if you're curious how this works, this is a good resource:
https://www.rabbitmq.com/tutorials/tutorial-six-python.html